admin-dev/themes/new-theme/js/pages/product/edit/product-form-mapping.js

```diff
@@ -26,19 +26,19 @@
 export default {
   'product.stock.quantity': [
     'product[stock][quantities][quantity]',
-    'product[shortcuts][stock][quantity]',
+    // 'product[shortcuts][stock][quantity]',
```

Comment:
Why do you comment this lines ?

Reply:
Because there is no shortcut column anymore in the new UX, as I explained in the description of this PR for now it is more "disabled" than completely removed, so the code has not been removed yet

This JS code caused bugs in the console because the shortcuts fields are not present anymore so I just commented this part, it will be removed along with the other code related to shortcut later but it doesn't mean this first modifications can't be merged

Once the page is more mature and Tristan validates the new display I will create a dedicated PR to clean everything correctly

Reply:
:+1: